doc/conf.py

```diff
@@ -1,13 +1,34 @@
+# This file is part of ts_MTDome.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
 """Sphinx configuration file for an LSST stack package.
 
 This configuration only affects single-package Sphinx documentation builds.
 """
 
-from documenteer.sphinxconfig.stackconf import build_package_configs
-import lsst.ts.MTDome
-
+from documenteer.conf.pipelinespkg import *  # noqa
+import lsst.ts.salobj  # noqa
 
-_g = globals()
-_g.update(
-    build_package_configs(project_name="ts_MTDome", version=lsst.ts.MTDome.__version__)
-)
+project = "ts_salobj"
```

Comment:
salobj -> MTDome

Reply:
Thanks for catching that. I changed it to the correct project name.